plugin-server/src/types.ts

```diff
@@ -140,6 +140,8 @@ export interface PluginsServerConfig extends CdpConfig, IngestionConsumerConfig
     TASK_TIMEOUT: number // how many seconds until tasks are timed out
     DATABASE_URL: string // Postgres database URL
     DATABASE_READONLY_URL: string // Optional read-only replica to the main Postgres database
+    PERSONS_DATABASE_URL: string // Optional read-write Postgres database for persons
+    PERSONS_READONLY_DATABASE_URL: string // Optional read-only replica to the persons Postgres database
```

Comment:
```suggestion
    PERSONS_DATABASE_URL?: string // Optional read-write Postgres database for persons
    PERSONS_READONLY_DATABASE_URL?: string // Optional read-only replica to the persons Postgres database
```

Reply:
I seem to recall doing something like that in the past and it not working. I left it as empty strings and then all the connection setup logic deals with the empty values correctly.